public_suffix_list.dat

```diff
@@ -11271,6 +11271,9 @@ zakopane.pl
 pantheon.io
 gotpantheon.com
 
+// Peplink | Pepwave : http://peplink.com/
+mypep.link
```

Comment:
Sorry for replying late, for some reason the email notification dropped to my junk box :(

I'm working on the update now, thank you :)